doc/source/input_output.rst

```diff
@@ -32,7 +36,26 @@ Output files and directory organization
 
     Files:
 
-        * Rate-adjustment results in tab-separated format: raw results for each trio (``adjustment_table_species_all.tsv``) and final results for each divergent pair after finding a consensus value in case of multiple outgroups (``adjustment_table_species.tsv``).
+        *   Raw rate-adjustment results for each trio (``adjustment_table_species_all.tsv``). Tabular format.
+
+            Each row shows the result for a species pair (column 2 ``Focal_Species`` and 3 ``Sister_Species``) diverging at a certain node (column 1 ``Node``) and adjusted with the outgroup in column 3 ``Out_Species``. The rate-adjusted mode with associated standard deviation are given in column 4 ``Adjusted_Mode`` and 5 ``Adjusted_Mode_SD``; for comparison it is provided the unadjusted mode with associated standard deviation (column 6 ``Original_Mode`` and 7 ``Original_Mode_SD``). The branch-specific *K*:sub:`S` contributions for the divergent species pair are listed in column 8 ``Ks_Focal`` and 9 ``Ks_Sister``; the *K*:sub:`S` contribution of the outgroup is listed in column 10 ``Ks_Out``.
```

Comment:
```suggestion
            Each row shows the result for a species pair (column 2 ``Focal_Species`` and 3 ``Sister_Species``) diverging at a certain node (column 1 ``Node``) and adjusted with the outgroup in column 3 ``Out_Species``. The rate-adjusted mode with associated standard deviation are given in column 4 ``Adjusted_Mode`` and 5 ``Adjusted_Mode_SD``; for comparison the unadjusted original mode with associated standard deviation is provided in column 6 ``Original_Mode`` and 7 ``Original_Mode_SD``. The branch-specific *K*:sub:`S` contributions for the divergent species pair are listed in column 8 ``Ks_Focal`` and 9 ``Ks_Sister``; the *K*:sub:`S`distance of the outgroup to the divergence event of the species pair is listed in column 10 ``Ks_Out``.
```

Reply:
I was afraid that "Ks distance" could be understood as an actual Ks estimate, while "contributions" indicates only the substitutions on the single branch.

Reply:
I understand but here it's not really a contribution, Ks_Out is the distance C to divergence of A and B (what we call O) not to the divergence of A and C or B and C.

Reply:
Ooh, I see... Let's then leave distance!